src/list/components/item-fields.js

```diff
@@ -96,18 +103,13 @@ export class EditItemFields extends React.Component {
 
     return (
       <div className={styles.itemFields}>
-        <label>
-          <Localized id="item-fields-title">
-            <LabelText className={styles.firstLabel}>tITLe</LabelText>
-          </Localized>
-          <FieldText data-name="title">{fields.title}</FieldText>
-        </label>
         <label>
           <Localized id="item-fields-origin">
-            <LabelText>oRIGIn</LabelText>
+            <LabelText className={styles.firstLabel}>oRIGIn</LabelText>
           </Localized>
           <Localized id="item-fields-origin-input" attrs={{placeholder: true}}>
             <Input className={styles.input} type="text"
+                   disabled={!!fields.itemId}
                    placeholder="wWw.eXAMPLe.cOm"
```

Comment:
The designs call for changing this to `"https://www.example.com" to match what we expect from the user.